packages/serve/src/lib/middleware/accessLogMiddleware.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  getLogger,
+  LoggerOptions,
+  VulcanInternalExtension,
+} from '@vulcan-sql/core';
+import * as bytes from 'bytes';
+import { BuiltInMiddleware, KoaContext, Next } from '@vulcan-sql/serve/models';
+
+@VulcanInternalExtension('access-log')
+export class AccessLogMiddleware extends BuiltInMiddleware<LoggerOptions> {
+  private logger = getLogger({
+    scopeName: 'ACCESS_LOG',
+    options: this.getOptions(),
+  });
+
+  public async handle(context: KoaContext, next: Next) {
+    if (!this.enabled) return next();
+
+    const { request: req, response: resp, params } = context;
+
+    const reqSize = req.length ? bytes(req.length).toLowerCase() : 'none';
+    const respSize = resp.length ? bytes(resp.length).toLowerCase() : 'none';
+    /**
+     * TODO: The response body of our API server might be huge.
+     * We can let users to set what data they want to record in config in the future.
+     */
```

Comment:
Should we remove this note?

Reply:
Thanks for finding the non removed comment, it has been removed, please check it